pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -53,16 +54,25 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(flowcontrol.GroupName, legacyscheme.Scheme, legacyscheme.ParameterCodec, legacyscheme.Codecs)
 
 	if apiResourceConfigSource.VersionEnabled(flowcontrolv1alpha1.SchemeGroupVersion) {
-		flowControlStorage, err := p.v1alpha1Storage(apiResourceConfigSource, restOptionsGetter)
+		v1alpha1FlowControlStorage, err := p.v1alpha1Storage(apiResourceConfigSource, restOptionsGetter)
 		if err != nil {
 			return genericapiserver.APIGroupInfo{}, false, err
 		}
-		apiGroupInfo.VersionedResourcesStorageMap[flowcontrolv1alpha1.SchemeGroupVersion.Version] = flowControlStorage
+		apiGroupInfo.VersionedResourcesStorageMap[flowcontrolv1alpha1.SchemeGroupVersion.Version] = v1alpha1FlowControlStorage
 	}
+	v1beta1FlowControlStorage, err := p.v1beta1Storage(apiResourceConfigSource, restOptionsGetter)
+	if err != nil {
+		return genericapiserver.APIGroupInfo{}, false, err
+	}
+	apiGroupInfo.VersionedResourcesStorageMap[flowcontrolapisv1beta1.SchemeGroupVersion.Version] = v1beta1FlowControlStorage
 	return apiGroupInfo, true, nil
 }
 
 func (p RESTStorageProvider) v1alpha1Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
+	return p.v1beta1Storage(apiResourceConfigSource, restOptionsGetter)
```

Comment:
should this call the v1beta1 function? if v1beta1 changes in the future, shouldn't the alpha storage function remain the same?

Reply:
Storage version should be v1beta1, so I'm not sure if we should have an alpha version of this function at all any more.

Reply:
fixed